### ISSUE TITLE
41560855 - Call of Duty: Black Ops

### ISSUE THREAD

**Issue by marinesciencedude** [xenia-canary/game-compatibility#98]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-canary/xenia-canary/wiki/FAQ).
- [X] The Xenia build used is from a canary branch. (not MLBS/AlexVS/master/pull requests, etc)
- [X] This issue isn't for help (tech support) with Xenia, or a game.
- [X] I've checked if this game doesn't already have a compatibility report.

### Xbox 360 Marketplace link

https://marketplace.xbox.com/en-us/Product/Call-of-Duty-Black-Ops/66acd000-77fe-1000-9115-d80241560855

### Xenia version

https://github.com/xenia-canary/xenia-canary/commit/5a71b552332f705f7b67c6ea5cd29d87b4fa502e

### Issues

Tendency to crash when going into gameplay on the first campaign mission (could be related to intro cinematic skipping)
Minor light sources glow through the map.

### Log

Load into gameplay crash: [xenia.zip](https://github.com/xenia-canary/game-compatibility/files/8772725/xenia.zip)
Main gameplay test (80.4MB archive, 1.37GB uncompressed): [xenia.zip](https://drive.google.com/file/d/1UxaTRkv6nFP8ODYsN4vN1vbbHnyhzGt-)

### Screenshots

![mainmenu](https://user-images.githubusercontent.com/14031281/170345069-f713b5cf-079e-457a-b742-e9b9c65edd2d.png)
![firstlevel](https://user-images.githubusercontent.com/14031281/170345074-27f05b38-f437-4260-afb7-3d12b631fb05.png)
![finalpic](https://user-images.githubusercontent.com/14031281/170345082-201733c8-e345-4d6b-9bae-80b384df24e3.png)

### Video

https://youtu.be/FbBYOYuzlH4

### Labels

state-gameplay, tech-engine-iw

**Comment by CrazyCSIW6** [comment]
Some additional notes:
1. Bloom effects appear to be missing compared to actual Xbox 360 hardware
2. Tendency to freeze after going into campaign or zombies menus on newer Canary builds
3. On older Canary builds, the M16 crashes the multiplayer [appears to have since been fixed]

**Comment by marinesciencedude** [comment]
### Xenia version
https://github.com/xenia-canary/xenia-canary/commit/9992d20079b34c28ae092df676ef6546ee8ce30e

### Notes
Loads Title Updates and DLC with `protect_zero = false`
Game froze upon reaching the second round in Ascension, need to test on a more powerful system.

### Log
[xenia.zip](https://github.com/xenia-canary/game-compatibility/files/11325892/xenia.zip)

### Screenshots
![DLC_ZombieMaps_Selection](https://user-images.githubusercontent.com/14031281/234376499-5705b53e-f2ac-450e-b34b-f9071c721d3d.png)
![AscensionLoadingScreen](https://user-images.githubusercontent.com/14031281/234376526-62ad14c8-363f-48af-b487-dee534e94359.png)
![AscensionStart](https://user-images.githubusercontent.com/14031281/234376555-0142f77e-fc76-4158-aae0-e14be7977965.png)
![AscensionBnW](https://user-images.githubusercontent.com/14031281/234376577-289bed21-6e36-4734-a29b-7e365d67232e.png)
![AscensionHang](https://user-images.githubusercontent.com/14031281/234376587-c359789d-5f71-43dd-83e6-f9c2252fae4a.png)

Final frame on which the game hanged.

**Comment by marinesciencedude** [comment]
### Xenia version
https://github.com/xenia-canary/xenia-canary/commit/94759861c52211acee9249ec0ce19528b2086734

### Issues
Crashes on Moon some time after starting the first round, otherwise good work by Adrian#5614 and Gliniak#3663 on getting it to work without `protect_zero`

### Log
[xenia.zip](https://github.com/xenia-canary/game-compatibility/files/11332618/xenia.zip)

### Screenshots
![MoonSelection](https://user-images.githubusercontent.com/14031281/234571087-54f4575d-b7a5-4df5-9938-c3ff5b640b38.png)
![MoonLoading](https://user-images.githubusercontent.com/14031281/234571102-83e5e8b2-6c0d-4e7c-a423-80e032c05d3a.png)
![NoMansLandStart](https://user-images.githubusercontent.com/14031281/234571120-ee409d13-9b34-46a8-a67a-3f4e11c5fbd0.png)
![GriffinStationStart](https://user-images.githubusercontent.com/14031281/234571193-5161a977-0d6c-45e0-b71d-0686a47351f7.png)
![GuestCrash](https://user-images.githubusercontent.com/14031281/234571203-5cd94047-fe71-4489-ad95-c5dcf2fe9dfd.png)

**Comment by Tendertitty** [comment]
Still has issues getting past the main menu, will consistently crash on the newest version of Canary.

**Comment by ghost** [comment]
Yup same here, seems to be a regression. Had it working in the past but now mostly crashes on the main menu / first mission.

**Comment by ros-j** [comment]
> Still has issues getting past the main menu, will consistently crash on the newest version of Canary.

Same here

**Comment by DaniProZ** [comment]
<img width="1919" height="1079" alt="Image" src="https://github.com/user-attachments/assets/1b2139fd-4077-4e82-8474-68b4bbde384b" /> The game always crash in the main menu

**Comment by BorealisEcho** [comment]
Tested on 30ac9d7_canary_experimental,

Finished the campaign without crashes or other major issues.
- Specs: RTX 5070ti / Ryzen 9800X3D / Win11

- Settings: `occlusion_query = "fast"`,  `readback_resolve = "fast"`

### Issues:

Sound:
- No issues

Visual:
- Lighting on character models flicker randomly from time to time. readback seems to kinda improve it somewhat but doesn't solve it.
- Needs `occlusion_query = "fast"` to fix unoccluded lens flares.

Gameplay:
- No issues, some stuttering from heavy shader compilation in the first level but it improves later on.
- Crashing is fixed.

### Screenshots:

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/8f4f3758-908e-4a3c-b3ee-9b145045c63e" />

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/9c2865de-274e-41ed-8dde-6877e558c364" />

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/c3754b36-9bd6-42c6-839e-c9a48ebc69d6" />

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/b63d0660-0aac-42af-84bb-989cb25b5ba2" />

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/e9948398-cd45-436e-8c62-9d9292e4176b" />

**Comment by Chorniy-Kovbas** [comment]
Tested on 0e1307b, completed 1/3 of the game, and unfortunately it's poorly playable in its current state. There are too many sudden FPS drops and stuttering during explosions or heavy gunfire. I don't recommend playing it in its current state — it needs proper optimization. Specs: Rtx 4060ti, ryzen 5 3600, 32 gb, win11.

**Comment by Marmatica** [comment]
I don't know what's wrong with your end but it runs fine for me once shaders have been compiled
